### PR TITLE
CherryPy consolidated the wsgiserver package in 6.1.0

### LINKED ISSUE #2109: cherrypy >= 6.1.0 fails tests
From the cherrypy [changelog]:

```
6.1.0
-----

* Combined wsgiserver2 and wsgiserver3 modules into a
  single module, ``cherrypy.wsgiserver``.
```

[changelog]: https://github.com/cherrypy/cherrypy/blob/master/CHANGES.txt

### PR DESCRIPTION
**Pull request by jstewmon**
fixes #2109

### CODE REVIEW

File: tests/functional/test_libraries.py [hunk 1]
```diff
@@ -68,8 +68,8 @@ def test_cherrypy(pyi_builder):
     # import wsgiserver3 on Python 3, or else wsgiserver2 on Python 2.
```
Comment (codewarrior0) [inline tests/functional/test_libraries.py]: This comment is now stale, mind snipping it out?

Comment (jstewmon) [inline tests/functional/test_libraries.py]: Ah, good catch. Snipped!

**Comment by jstewmon** [comment]
The build is failing due to a numpy issue...

**Comment by htgoebel** [comment]
Thanks for the pull-request. However I'd ask you to amend the commit message into something like:

> Remove hook cherrypy
> 
> In version 6.1.0 CherryPy consolidated the wsgiserver package, so the
> hook is not longer used.

Thanks.

**Comment by htgoebel** [comment]
@jstewmon Thanks for amending the commit message :-)

**Pull request merged**